### ISSUE TITLE
Als medewerker wil ik een aanvraag Eigen Locatie kunnen starten vanuit een reservering in het medewerkerdasboard

### ISSUE THREAD

**Issue by MichelleDautzenberg** [VNG-Realisatie/huwelijksplanner#528]
zodat ook dit proces aan de balie gestart kan worden door een medewerker.

Hoort ook bij #484 

- [ ] Zorg dat de beschrijving duidelijk voor iedereen in het team is
- [ ] Koppel de juiste labels
- [ ] Koppel eventueel aan een epic (door # en het nummer toe te voegen)
- [ ] Voeg alle taken die nodig zijn voor de user story toe.

**Comment by rubenvdlinde** [comment]
Zelfde BPM proces als voor de burger?

**Comment by MichelleDautzenberg** [comment]
Ja, zelfde BPMN proces. Gaat er hier om dat een inwoner die aan de balie komt hiervoor kan kiezen en dat de medewerker dit dus vanuit medewerkerportaal in gang moet kunnen zetten.

**Comment by MichelleDautzenberg** [comment]
Hiervoor geldt dus ook dat door de medewerker vanuit het medewerkersportaal dezelfde velden in OpenZaak gevuld moeten worden als vanuit het inwonersformulier.

Mapping van formuliervelden naar OpenZaak:
Eigenschap 1) locatieAdres --> samenvoeging van velden straat en huisnummer(toevoeging), postcode, plaats.
Eigenschap 2) locatieNaam --> veld locatienaam
Eigenschap 3) locatieTelefoon --> veld telefoonnummer locatie
Eigenschap 4) locatieOmschrijving --> veld omschrijving locatie

Zaakobject 1) verwijzing naar de bijbehorende reservering in de huwelijksplanner.
